source/base/mpi.inst.in

```diff
@@ -14,6 +14,8 @@
 // ---------------------------------------------------------------------
 
 
+#ifndef DOXYGEN
+
```

Comment:
As far as I remember, you need to write the macro inside of the for-loop.

Reply:
you are correct. I decided to teach the tool to understand it.